input/v1beta1/input.go

```diff
@@ -19,6 +19,10 @@ type GoTemplate struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Template start characters
+	LeftDelims string `json:"leftDelims,omitempty"`
+	// Template end characters
+	RightDelims string `json:"rightDelims,omitempty"`
```

Comment:
```suggestion
	// Template start characters
	// +optional
	LeftDelims *string `json:"leftDelims,omitempty"`
	// Template end characters
	// +optional
	RightDelims *string `json:"rightDelims,omitempty"`
	
```

These are optional, right? Even though this isn't a real CRD we try to follow CRD conventions. So for optional fields we want them to be pointers, and have the `+optional` comment marker.